image-layout.md

```diff
@@ -156,4 +161,48 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## index.json
+
+This OPTIONAL file is for use indexing references and elements of the image-layout, where walking the `refs` and `blobs` is sub-optimal.
```

Comment:
The index file has no effect on whether you need to or can walk `blobs`.  Performance issues aside, it is a drop-in replacement for `refs`, except that [the previous ref names are gone][1].  So I think we should remove the `blobs` reference from this line.

[0]: https://github.com/opencontainers/image-spec/pull/438#issue-187078763
[1]: https://github.com/opencontainers/image-spec/pull/438#issuecomment-264035961